stripe/src/main/java/com/stripe/android/model/ExpirationDate.kt

```diff
@@ -54,6 +54,10 @@ sealed class ExpirationDate {
             ).joinToString(separator = "")
         }
 
+        private val isComplete = (month.length + year.length) == 4
+
+        internal val isIncomplete = !isComplete && ((month.length + year.length) > 0)
```

Comment:
I guess it's obvious from looking at the code, but it might be helpful to explain why `isIncomplete` is not just `!isComplete`

Reply:
Might be better to name it, isPartial, or something like that?